Makefile

```diff
@@ -106,3 +106,21 @@ proto: install-tools ## Generate protobuf files
 .PHONY: bench-kafka
 bench-kafka:
 	go test -count 1 -run BenchmarkCompression -bench=. -benchmem ./services/streammanager/kafka/client
+
+.PHONY: check-swagger
+check-swagger:
+	which swagger || (GO111MODULE=off go get -u github.com/go-swagger/go-swagger/cmd/swagger)
+
+.PHONY: swagger
+swagger: check-swagger
+	GO111MODULE=on go mod vendor  && GO111MODULE=off swagger generate spec -o ./swagger.yaml --scan-models
+
+.PHONY: serve-swagger
+serve-swagger: check-swagger
+	go install github.com/go-swagger/go-swagger/cmd/swagger
+	swagger serve ./swagger.yaml
+
+.PHONY: serve-swagger-ui
+serve-swagger-ui: check-swagger
+	go install github.com/go-swagger/go-swagger/cmd/swagger
```

Comment:
We can probably move this to install-tools.

Reply:
`check-swagger` is not working. This would be sufficient in Makefile.
```GO
.PHONY: serve-swagger-ui
serve-swagger-ui: install-tools
	swagger validate ./swagger.yaml
	swagger serve -F swagger ./swagger.yaml
```

Reply:
Done.

Reply:
You would need to add `go install github.com/go-swagger/go-swagger/cmd/swagger` inside install-tools. Also, we can probably remove